src/coreclr/jit/gentree.h

```diff
@@ -6249,11 +6249,13 @@ struct GenTreeHWIntrinsic : public GenTreeJitIntrinsic
     bool OperIsEmbBroadcastCompatible() const;
     bool OperIsBroadcastScalar() const;
     bool OperIsCreateScalarUnsafe() const;
+    bool OperIsBitwiseHWIntrinsic();
```

Comment:
Shouldn't this be `bool OperIsBitwiseHWIntrinsic() const;` to indicate it doesn't mutate the instance?

Reply:
Yes, it should be annotated as such since it doesn't mutate (and never should)